library/model/book.py

```diff
@@ -7,6 +7,12 @@
 
 from library.persistence.storage import LibraryRepository
 
+time_per_page_paper = 3 # in mins
```

Comment:
We think it is a good idea to separate these values into different variables. 
But we are not sure if it is a good option to put them as global, maybe we can have them in a Config file/ class.